docker.go

```diff
@@ -26,6 +27,15 @@ var (
 		RunE:         runDockerPin,
 	}
 
+	dockerBaseCmd = &cobra.Command{
+		Use:          "resolve [base image]",
```

Comment:
Let's make this `[base-image]` so people/Cobra understand this is a single argument.